lua/lua-core/intro-to-lua-ii/installing-lua-iv.md

```diff
@@ -0,0 +1,23 @@
+---
+author: Stefan-Stojanovic
+category: must-know
+type: normal
+links:
+  - >-
+    [Download VSCode](https://code.visualstudio.com/download){website}
+  - >-
+    [Download Sublime](https://www.sublimetext.com/3){website}
+
+---
+
+# Installing Text Editors
+
+---
+
+## Content
+
+As for text editors, there are many options to choose from, including **Sublime Text**, **Atom**, **Notepad++**, **VSCode** and more. 
```

Comment:
```suggestion
As for text editors, there are many options to choose from, including **Sublime Text**, **Notepad++**, **VSCode** and more. 
```

Remove  **Atom** as it was deprecated last year or the year before that

Reply:
Should be noted that insights 2, 3, 4 will be removed and insight 1 will be moved into the first workout, effectively deprecating the installation workout.

This will be carried out immediately after this PR merges through the arching Lua workout PR.